dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/filter/WebAppFilterManager.java

```diff
@@ -1177,7 +1178,8 @@ else if (isTraceOn && logger.isLoggable(Level.FINE)) {
                                 alreadyVerifiedEncodedChar = true;        //to skip check in filterDefined later on
                             }
                             catch (IOException ioe) {
-                                logger.logp(Level.FINE, CLASS_NAME, "invokeFilters", "genericServletWrapper. Bad request - sending 400 [" + ioe.getMessage() + "]");
+                                if (isTraceOn && logger.isLoggable(Level.FINE)) 
```

Comment:
should it be Level.FINER?

Also, we always request `all` level for WC must gather.  Does Level.FINE or FINER make any difference?

Reply:
I have it be FINE in isLoggable to match the FINE in the logp call.  The isLoggable FINER updates in this class are for the entering and exiting calls because those are FINER calls.  entering and exiting will do a isLoggable check for FINER as the first thing before actually trying to log.  As to you requesting `all` for WC must gather, you are correct that ti doesn't make any difference in that case.  I just was doing some cleanup for the FINER changes since I was in the file so I figured I might as well do those ones right as well.